### DISCUSSION TITLE
Flatpack/ Installation in fedora?

### DISCUSSION THREAD

**Discussion by Joao-kouznetz** [ActivityWatch/activitywatch#996] [Q&A]
I use Fedora Linux and I'm having difficulty installing the app. Can someone explain to me how to download it? And to make it easier to use the app, is there any schedule to release it in Flatpak?

**Comment by 2e3s** [comment]
https://github.com/ActivityWatch/activitywatch/issues/476
Not any time soon. Please, use other ways to use the app. Here are the links https://github.com/ActivityWatch/activitywatch/releases/tag/v0.12.2 the easiest way is to unpack the zip somewhere and run from there.
